### DISCUSSION TITLE
关于pc端的规则打不开的问题

### DISCUSSION THREAD

**Discussion by yunyan0228** [xiaohucode/yidaRule#116] [Q&A]
需要挂梯子的规则，在安卓端都可以正常打开
但是在pc端就打不开，提示如下
![image](https://github.com/xiaohucode/yidaRule/assets/119416578/ab29935a-8dfa-4b42-ac2f-85ae4d4c94d2)

**Comment by xiaohucode** [comment]
开启 tun 模式或者在 app 里设置代理
